app.js

```diff
@@ -2,7 +2,116 @@ const toggle = document.querySelector('.nav__links');
+const projectItems = [
+  {
+    id: 1,
+    img: 'Image 1',
+    title: 'Multi-Post Stories Gain+Glory 1',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 2,
+    img: 'Image 2',
+    title: 'Multi-Post Stories Gain+Glory 2',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 3,
+    img: 'Image 3',
+    title: 'Multi-Post Stories Gain+Glory 3',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html', 'git'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 4,
+    img: 'Image 4',
+    title: 'Multi-Post Stories Gain+Glory 4',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 5,
+    img: 'Image 5',
+    title: 'Multi-Post Stories Gain+Glory 5',
+    technologies: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus sed labore maxime neque, omnis at soluta eius quo non, dolorum amet a fuga aspernatur aliquam modi odit, tempore perspiciatis architecto! Assumenda nostrum veniam ab cupiditate, error quidem numquam voluptas? Consequatur alias repudiandae architecto beatae doloremque dolor sit, eveniet delectus corrupti.',
+    buttonText: 'See Project',
+  },
+  {
+    id: 6,
+    img: 'Image 6',
```

Comment:
Kindly use a proper image url for each of the projects in order to have the images displayed and to fully represent the design template.